pdfgeneration/pdfgeneration.py

```diff
@@ -174,9 +174,13 @@ def run(self, options):
             # replace the values
             txt = txt.replace("${PATIENT_TOKEN}", options.patientId)
             txt = txt.replace("${PREDICTION_CLASSIFICATION}", classification_data['prediction'])
-            txt = txt.replace("${COVID-19}", classification_data['COVID-19'])
-            txt = txt.replace("${NORMAL}", classification_data['Normal'])
-            txt = txt.replace("${PNEUMONIA}", classification_data['Pneumonia'])
+            
+            predictionAnalysis = ""
```

Comment:
Variable names should be lowercase, with words separated by underscores as necessary to improve readability.

```suggestion
            prediction_analysis = ""
```

You can find guidelines and best practices of coding in Python in the
[PEP 8 documentations](https://www.python.org/dev/peps/pep-0008/).